client/src/components/hpaPanel.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Field from './field';
+
+const HpaPanel = ({spec}) => (
+    <>
```

Comment:
Small nit-pick. I believe you can remove most/all of the `<>` tags below. Those only required if you want to return more than one root level tag. So needed here...
```
<>
  <div />
  <div />
</>
```
But not needed here...
```
<div>
  <div />
  <div />
</div>
```